utils/spacewalk-common-channels.ini

```
@@ -1296,6 +1296,46 @@ gpgkey_id = %(_uyuni_gpgkey_id)s
 gpgkey_fingerprint = %(_uyuni_gpgkey_fingerprint)s
 repo_url = https://download.opensuse.org/repositories/systemsmanagement:/Uyuni:/Master:/openSUSE_Leap_15-Uyuni-Client-Tools/openSUSE_Leap_15.0/
 
+[opensuse_micro5_4]
+checksum = sha256
+archs    = x86_64, aarch64
+name     = openSUSE Leap Micro 5.4 (%(arch)s)
+gpgkey_url = https://download.opensuse.org/distribution/leap-micro/5.4/product/repo/Leap-Micro-5.4-x86_64-Media1/gpg-pubkey-3dbdc284-53674dd4.asc
+gpgkey_id = 3DBDC284
+gpgkey_fingerprint = 22C0 7BA5 3417 8CD0 2EFE  22AA B88B 2FD4 3DBD C284
+repo_url = https://download.opensuse.org/distribution/leap-micro/5.4/product/repo/Leap-Micro-5.4-%(arch)s-Media1/
+dist_map_release = 15.5
+
+[opensuse_micro5_4-sle-updates]
+label    = %(base_channel)s-sle-updates
+name     = SLE Micro Update Repository (%(arch)s)
```

his will cause conflicts. Use:
```suggestion
name     = SLE Micro 5.4 Update Repository (%(arch)s)
```

---

See https://github.com/uyuni-project/uyuni/pull/6830 for reference.